<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddLibraryDialog</class>
 <widget class="QDialog" name="AddLibraryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="outer_layout">
   <item>
    <layout class="QGridLayout" name="upper_grid_layout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Library Path</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="path_field">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="open_btn">
       <property name="text">
        <string>Open Library</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="add_library_btn">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Add Library</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>880</width>
        <height>488</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="scroll_layout_dummy">
       <item>
        <widget class="QTableWidget" name="series_table">
         <property name="showGrid">
          <bool>true</bool>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderHighlightSections">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Title</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Site ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>My Volumes</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>English Volumes</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Source Volumes</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="bottom_btn_layout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="done_btn">
       <property name="text">
        <string>Done</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_btn">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
